content/Solutions/MigratingToTrueNAS.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Migrating to TrueNAS"
+description: "This article describes general recomendations for migrating data into TrueNAS."
+weight: 10
+tags:
+- coregettingstarted
+---
+
+{{< toc >}}
+
+Every NAS user has a unique storage setup, but we can still give general recommendations for migrating your data into TrueNAS using share protocols and cloud storage.
+
+### NFS Migration
+
+If you are using computers with Linux OS (or a Unix-like OS), we recommend using [rsync](https://rsync.samba.org/) to transfer files from your current NAS storage into TrueNAS. Rsync is an open-source file transfer utility. You can find usage instructions and tutorials on the rsync website.
+
+{{< hint info >}}
+If you decide to use rsync, be aware that your filesystem must support Access Control Lists (ACLs).
+{{< /hint >}}
+
+### SMB Migration
+
+We recommend migrating via SMB sharing if you use computers with Windows OS (or if you prefer SMB). 
+
+[Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) is ideal for users with Windows clients. SMB also allows you to drag & drop files to migrate them from your current NAS storage into TrueNAS, but it will be slower than using a utility like Robocopy.
```

Comment:
[Robocopy](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) is ideal for users with Windows clients. SMB also allows you to drag & drop files to migrate them from your current NAS storage into TrueNAS. Using SMB will be slower than using a utility like Robocopy.
```